### PR TITLE
[BIP-158] Migrate wstETH/WETH Gauge [Arbitrum] 

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

File: BIPs/BIP-158/mainnetDAOMultisig.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "description": "",
+    "txBuilderVersion": "1.11.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x6ea01ca34c154eabb0657209964f6f59791a6ea521b652273b7790c5934d1fd7"
+  },
+  "transactions": [
+    {
+      "to": "0x8F42aDBbA1B16EaAE3BB5754915E0D06059aDd75",
+      "value": "1000000000000000000",
```
Comment (Tritium-VLK) [inline BIPs/BIP-158/mainnetDAOMultisig.json:16]: I am unsure why this value is not 0.

File: BIPs/BIP-158/mainnetGaugeMultisig.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "address", "name": "rootGauge", "type": "address" }
+        ],
+        "name": "addArbitrumGauge",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "rootGauge": "0x519cCe718FCD11AC09194CFf4517F12D263BE067"
```
Comment (Tritium-VLK) [inline BIPs/BIP-158/mainnetGaugeMultisig.json:26]: This gauge has a recipient set that leads to an EOA.  Is this correct?

Comment (Tritium-VLK) [reply BIPs/BIP-158/mainnetGaugeMultisig.json:26]: Note that the old gauge also has an EOA as recipient.  Maybe I'm missing something?

**Comment by Tritium-VLK** [comment]
@solarcurvey Zen put this PR together.  I restructured it and did an initial review, my first one because I did the other side of the rest of them.  You mind taking a look at the review comments here?  I think I'm missing something.  Would also be good to understand everything you check in review.

File: BIPs/BIP-158/mainnetGaugeMultisig.json (hunk 2 above)
Comment (markusbkoch) [reply BIPs/BIP-158/mainnetGaugeMultisig.json:26]: The recipient lives in the L2 chain associated with the root gauge. In this case, https://arbiscan.io/address/0xBAF1eE3D36347Fab45AcB987B9606dDbE99Ba5D0#code

File: BIPs/BIP-158/mainnetDAOMultisig.json (hunk 1 above)
Comment (solarcurvey) [reply BIPs/BIP-158/mainnetDAOMultisig.json:16]: yep value should be 0

**Comment by Tritium-VLK** [comment]
https://snapshot.org/#/balancer.eth/proposal/0x208760fac7a90fc6955e0ea234782fa965121fbf344ada8178f237d73ac88463

**Pull request merged**